### DISCUSSION TITLE
ikuai使用nat过滤后，wan2线路异常

### DISCUSSION THREAD

**Discussion by HomeQi** [joyanhui/ikuai-bypass#34] [Q&A] [answered]
<img width="519" alt="image" src="https://github.com/user-attachments/assets/aa3d6789-cb42-43c3-ad4b-44e91dce516d">
尝试减少一层nat，目的是方便OpenWrt中对流量来源的控制，但是使用后会导致wan2无法正常使用，请教大家，十分感谢！

**Comment by joyanhui** [comment]
知识点盲区，等下一位

**Comment by refined-fish** [comment, marked as answer]
我认为是你的配置问题，因为我也是这样过滤的，以便openwrt的clash面板获取真实的客户端IP，减少NAT，需要提供更多信息以便排查
